docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: GX Overview
```

Comment:
```suggestion
title: Great Expectations Overview
```
It would be great if we could move away from these types of titles (Overview). Perhaps _Great Expectations fundamentals_,  _Great Expectations basics_,  _An introduction to Great Expectations_, _Get to know Great Expectations_? The title should be more informative and inviting. 

We should use Great Expectations on first use. This is first use. Also, I recommend defining the acronym in the body text on first use.  For example: Great Expectations (GX). GX can then be used after the acronym has been defined.